src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+
+import { TodoItem } from '../TodoItem/TodoItem';
+import { TodosContext } from '../TodoContext/TodoContext';
+
+export const TodoList = () => {
+  const { filterTodos, tempTodo, processingIds } = useContext(TodosContext);
+
+  return (
+    <section className="todoapp__main">
+      {filterTodos().map(todo => (
```

Comment:
please change the name of this function:
```suggestion
      {getFilteredTodos().map(todo => (
```